contracts/LivepeerProtocol.sol

```diff
@@ -196,6 +215,9 @@ contract LivepeerProtocol {
         // Fail no more than 1% of the time
         verificationFailureThreshold = 1;
 
+        // A job becomes inactive 100 blocks after endJob() is called
+        jobEndingPeriod = 100;
```

Comment:
This is a test value rather than an actual default value. Although in reality, the default value would be larger, our tests use a helper function to fast forward a certain number of blocks by repeatedly using TestRPC's `evm_mine` request. A large number of `evm_mine` requests slows down our tests a lot, so I think it makes sense to use smaller values for time related variables just for testing purposes. I can file an issue as a reminder to come back and replace these test values with realistic default values.